internal/cli/cluster/cluster.go

```diff
@@ -186,7 +186,14 @@ func (o *ObjectsGetter) Get() (*ClusterObjects, error) {
 		}
 		// get nodes where the pods are located
 	podLoop:
-		for _, pod := range objs.Pods.Items {
+		for i, pod := range objs.Pods.Items {
+			// filter back-up job pod
+			labels := pod.GetLabels()
+			if labels != nil && labels[constant.DataProtectionLabelBackupNameKey] != "" {
```

Comment:
do not need to check if map is nil for finding data.  
even if the map is nil, it will return an empty value according to struct type if not found.
but for setting value, nil value should check